OutOfSchool/OutOfSchool.WebApi/Services/AreaAdminService.cs

```diff
@@ -37,7 +37,7 @@ public class AreaAdminService : CommunicationService, IAreaAdminService
         ICurrentUserService currentUserService,
         IMinistryAdminService ministryAdminService,
         IRegionAdminService regionAdminService)
-        : base(httpClientFactory, communicationConfig?.Value, logger)
+        : base(httpClientFactory, communicationConfig, logger)
```

Comment:
(Design) Consider extracting base class - `CommunicationService` - as an injected dependency. 
Same for other siblings (`GeocodingService`, ...)

Reply:
We discussed this, but since this part will be reworked, I was told not to change it

Reply:
Thanks for info. Then let's postpone it